index.html

```diff
@@ -1,30 +1,36 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="h-screen">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css">
-    <link 
-    href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css" 
-    rel="stylesheet"  type='text/css'>
+    <link
+      href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <!-- <link href="./output.css" rel="stylesheet" /> -->
     <script src="./src/JS/home.js" defer></script>
     <title>Vite App</title>
   </head>
-  <body>
-    <div class="home-page">
-      <div class="logo-container"></div>
-      <div class="app-description">
-        <p class="highlighted-text"> 
-          Your go-to app for all your <span>karting</span> infomation.
-        </p>
-        <p class="sub-text">
-          On your marks, get set...
+  <body class="h-screen bg-primaryColor text-primaryTextColor font-sans">
+    <div
+      class="flex flex-col h-screen justify-evenly items-center"
+      class="text-secondaryColor"
```

Comment:
noticed you have two classes here, you can just combine them into one.

Reply:
Oh yes, let me do that.